src/xdocs/google_style.xml

```diff
@@ -2437,6 +2437,31 @@
           </table>
         </div>
       </subsection>
+      <subsection name="Overrides" id="Google_Overrides">
+        <p>
+          To change the default severity level, use the following custom property variable:
```

Comment:
`To change the default severity level, use the following system property variable:`